src/token_store.cc

```diff
@@ -85,11 +83,12 @@ int TokenStore::Refresh() {
       "&grant_type=refresh_token&refresh_token=" + refresh_;
   std::string response;
   long response_code = 0;
+  log_->Write("TokenStore::Refresh: token_endpoint: %s", Config::Get()->token_endpoint());
```

Comment:
%s in the format string expects a char*, so you'll need "Config::Get()->token_endpoint().c_str()" instead.